src/xdocs/checks/whitespace/genericwhitespace.xml.template

```diff
@@ -40,23 +40,15 @@
         </p>
         <macro name="example">
           <param name="path"
-                 value="resources/com/puppycrawl/tools/checkstyle/checks/whitespace/genericwhitespace/Example1.txt"/>
+                 value="resources/com/puppycrawl/tools/checkstyle/checks/whitespace/genericwhitespace/Example1.java"/>
           <param name="type" value="config"/>
         </macro>
         <p id="Example1-code">
-          Examples with correct spacing:
+          Examples with correct and uncorrect spacing:
```

Comment:
```suggestion
          Example:
```
`uncorrect` is an uncommon/unused word, and we have no need for extra text here regardless.

Reply:
solved